app/lib/features/settings/pages/email_addresses.dart

```diff
@@ -0,0 +1,244 @@
+            child: Text(
+              "You added these email addresses but didn't confirm them yet. Once you confirmed them please acknowledge it here and they will be added to your confirmed addresses.",
+              style: Theme.of(context).textTheme.bodyMedium,
+            ),
+          ),
+        ),
+        SliverList.builder(
+          itemBuilder: (BuildContext context, int index) {
+            return EmailAddressCard(
+              emailAddress: addresses.unconfirmed[index],
+              isConfirmed: false,
+            );
+          },
+          itemCount: addresses.unconfirmed.length,
+        ),
+      ];
+      if (addresses.confirmed.isNotEmpty) {
+        slivers.addAll([
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: const EdgeInsetsDirectional.symmetric(
+                horizontal: 20,
+                vertical: 15,
+              ),
+              child: Text(
+                'Confirmed Email Addresses',
+                style: Theme.of(context).textTheme.headlineSmall,
+              ),
+            ),
+          ),
+          SliverList.builder(
+            itemBuilder: (BuildContext context, int index) {
+              return EmailAddressCard(
+                emailAddress: addresses.confirmed[index],
+                isConfirmed: true,
+              );
+            },
+            itemCount: addresses.confirmed.length,
+          ),
+        ]);
+      }
+      return CustomScrollView(slivers: slivers);
+    }
+
+    return CustomScrollView(
+      slivers: [
+        SliverToBoxAdapter(
+          child: Padding(
+            padding: const EdgeInsetsDirectional.symmetric(
+              horizontal: 20,
+              vertical: 15,
+            ),
+            child: Text(
+              'Confirmed emails addresses connected to your account:',
+              style: Theme.of(context).textTheme.bodyMedium,
+            ),
+          ),
+        ),
+        SliverList.builder(
+          itemBuilder: (BuildContext context, int index) {
+            return EmailAddressCard(
+              emailAddress: addresses.confirmed[index],
+              isConfirmed: true,
+            );
+          },
+          itemCount: addresses.confirmed.length,
+        ),
+      ],
+    );
+  }
+
+  Future<void> addEmailAddress(BuildContext context, WidgetRef ref) async {
+    final client = ref.read(clientProvider);
+    final manager = client!.threePidManager();
+    final newValue = await showDialog<String>(
+      context: context,
+      builder: (BuildContext context) => const AddEmailAddr(),
+    );
+    if (newValue != null) {
+      EasyLoading.show(status: 'Requesting token via email');
+      try {
+        await manager.requestTokenViaEmail(
+          newValue,
+        );
+        ref.invalidate(emailAddressesProvider);
+        EasyLoading.showSuccess(
+          'Please check your inbox for the validation email',
+        );
+      } catch (e) {
+        EasyLoading.showSuccess(
```

Comment:
could use ```showError()``` method here for showing error log.

Reply:
this was actually from a previous PR that is now not part of this PR anymore as it is already in `main`... If you still want to discuss that, let's address it separately.